tiledb/sm/serialization/fragment_info.cc

```diff
@@ -302,6 +302,8 @@ Status fragment_info_from_capnp(
 
   // Get single_fragment_info from capnp
   if (fragment_info_reader.hasFragmentInfo()) {
+    fragment_info->single_fragment_info_vec().reserve(
+        fragment_info_reader.getFragmentInfo().size());
```

Comment:
Is this a bug fix?

Reply:
Just an optimization that I did by the way :-)